pom.xml

```diff
@@ -34,6 +34,50 @@
           <skipTests>true</skipTests>
         </configuration>
       </plugin>
+      
+      <plugin>
+    <groupId>net.wasdev.wlp.maven.plugins</groupId>
+    <artifactId>liberty-maven-plugin</artifactId>
+    <version>2.1.2</version>
+    <configuration>
+        <assemblyArtifact>
+            <groupId>io.openliberty</groupId>
```

Comment:
I see that you're using openliberty, but we should try to use payara micro. Could you please give it a try with payara micro ?